src/components/scrollToTop/ScrollToTop.jsx

```diff
@@ -0,0 +1,32 @@
+import React, {useState} from 'react';
+import { Fab } from '@material-ui/core';
+import ArrowUpIcon from '@material-ui/icons/ArrowUpward';
+import "./ScrollToTop.scss";
+
+const ScrollArrow = () =>{
+
+  const [showScroll, setShowScroll] = useState(false);
+
+  const checkScrollTop = () => {
+    (window.pageYOffset > 500) ? setShowScroll(true) : setShowScroll(false)
+  };
+    
+  const scrollToTop = () =>{
+    window.scrollTo({top: 0, behavior: 'smooth'});
+  };
+
+  window.addEventListener('scroll', checkScrollTop);
+
+  return (
+    <div className="scroll-to-top" onClick={scrollToTop}>
+      {showScroll && 
+      <div >
```

Comment:
nit - fix formatting and indentation

Reply:
I am not sure how to format here

Reply:
Instead of this
```
<div>
<Fab color="primary" aria-label="scroll back to top">
<ArrowUpIcon fontSize="large" />
</Fab>
</div>
```
more like this

```
<div>
  <Fab color="primary" aria-label="scroll back to top">
  <ArrowUpIcon fontSize="large" />
  </Fab>
</div>
```

You can also run the command `npm run lint:fix` to format everything

Reply:
thanks!